services/discord/discord-boosts.service.js

```diff
@@ -0,0 +1,76 @@
+import Joi from 'joi'
+import { nonNegativeInteger } from '../validators.js'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  guild: {
+    premium_subscription_count: nonNegativeInteger,
+  },
+}).required()
+
+const documentation = `
+<p>
+  The Discord badge requires the <code>SERVER INVITE CODE</code> in order access the Discord JSON API, this invite code should be set to never expire.
+</p>
+<p>
+  The <code>SERVER INVITE CODE</code> can be located at the end of the invite url.
+</p>
+`
+
+export default class Discord extends BaseJsonService {
```

Comment:
HI!
The `test-main` fails because all your classes have the same name (Discord) you need to provide unique name for all your classes in `.service` files (like DiscordBoosts, DiscordMembers...)